Core/include/Acts/Surfaces/RegularSurface.hpp

```diff
@@ -0,0 +1,98 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2016-2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Surfaces/Surface.hpp"
+#include "Acts/Utilities/ThrowAssert.hpp"
+
+namespace Acts {
+
+class RegularSurface : public Surface {
```

Comment:
```suggestion
/// A physical surface which does not depend on the direction you look at it from. As such it narrows the interface of @class Surface and allows inspection without providing a global position and direction.
class RegularSurface : public Surface {
```

should we add a small into to this class?